src/apiCalls.js

```diff
@@ -0,0 +1,18 @@
+import endpoints from './endpoints.js';
+
+export const postNewEmailSubscriber = (newEmailSubscriber) => {
+  const { first_name, last_name, state_name, email, language } = newEmailSubscriber;
+
+  return fetch(endpoints.submit, {
+    mode: 'cors',
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json', 'Accept': 'application/json' },
+    body: JSON.stringify({
+      first_name: first_name,
+      last_name: last_name,
+      state_name: state_name,
+      email: email,
+      language: language
+    })
```

Comment:
This information is already being captured in the form component, so we'd just pass in the argument.